pwa/src/components/handlers/handlerForm.tsx

```diff
@@ -20,6 +20,7 @@ import {MultiDimensionalArrayInput} from "../common/multiDimensionalArrayInput";
 import ElementCreationNew from "../common/elementCreationNew";
 import {navigate} from "gatsby-link";
 import LoadingOverlay from "../loadingOverlay/loadingOverlay";
+import {TextareaGroup} from "../common/textareaGroup";
```

Comment:
The ticket is regarding **Endpoint** forms and not this form. So I think this implementation is not in the right spot.

I understand the reasoning behind the change, **please check with @rubenvdlinde if this change is wanted as we did not refine it.**